docs/changelog.md

```diff
@@ -0,0 +1,174 @@
+# Changelog
+
+## [0.1.4]
+
+Only API change: added a deserialize_plotly option to read() to allow reading the Plotly JSON string.
+
+Added:
+
+- (Re)process runs from the GUI (!285).
+- Add support for optionally returning the Plotly JSON string (!328).
+
+Changed:
+
+- Check context file without blocking GUI thread (!291).
+- Defer some imports in damnit.api (!281).
+- Evaluate the context file in the database directory when validating (!301).
+- Update default Python for context file to 202402 environment (!315).
+- xarray plot cbar range limit (!312).
+- Refactor GUI plotting code (!316).
+
+Fixed:
+
+- reprocessing from the GUI when the context dir is not the CWD (!302).
+- Add back log message on successful variable computation (!303).
+- allow plotting variable with Cell's custom summary value (!306).
+- passing through --var options to cluster job (!320).
+
+[Full Changelog](https://github.com/European-XFEL/DAMNIT/compare/0.1.3...0.1.4)
+
+## [0.1.3]
+
+Added:
+
+- Log timing after each successful variable computation (!247).
+- Show tooltip with full size thumbnail for image items (!263).
+- Save plotly figure (!262).
+- Add an option to force software OpenGL (!268).
+- Templates for setting up new context files (!248).
+- Table cell formatting with a Cell class (!221).
+- Add a database config option to skip starting a listener (!277).
+- Add back --direct option to run reprocessing directly on local node (!287).
+- Plot xarray (!289).
+
+Changed:
+
+- Preserve context file path for tracebacks (!249).
+- Improve error handling when opening variables in the GUI (!257).
+- Evaluate non-cluster variables in Slurm using Solaris cluster (!256).
+- Use Slurm jobs for reprocessing (!270).
+- HDF5 array compression (!273).
+- Simplify testing CLI by passing argv in (!279).
```

Comment:
I generally think changelogs should describe user-visible changes, so I tend to leave out things about testing, CI, and most docs changes. YMMV :slightly_smiling_face:

Reply:
right, I did remove quite a bit already, some may have slip through. I cleaned a bit more now